### PR TITLE
manually add timedropper@1.0

### LINKED ISSUE #8103: [Request] Add timedropper
**Library name:** timedropper
**Git repository url:** https://github.com/felicegattuso/timedropper/
**npm package url(optional):** 
**License(s):** MIT License, CC BY 4.0
**Official homepage:**  http://felicegattuso.com/projects/timedropper/
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by kennynaoh**
PR for #8103 
@pvnr0082t please help me review this pull request, thank you.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/felicegattuso/timedropper
 * Official website (optional, not the repository):
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 753
   - Count of watchers: 23
   - Count of forks: 63
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #8103, cc @felicegattuso

### CODE REVIEW

File: ajax/libs/timedropper/package.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/felicegattuso/timedropper.git"
+  },
+  "keywords": [
+    "timedropper",
+    "timepicker"
+  ],
+  "author": "Felice Gattuso",
+  "license": "MIT"
```
Comment (pvnr0082t) [inline ajax/libs/timedropper/package.json]: The `LICENSE` section in http://felicegattuso.com/projects/timedropper/ shows that it's dual licenses. Could you confirm with author? @kennynaoh

**Comment by kennynaoh** [comment]
@pvnr0082t 
Dual license is correct.
And `homepage` added.
Please help me review it again.
Thanks!

**Comment by pvnr0082t** [comment]
LGTM

**Comment by PeterDaveHello** [comment]
@kennynaoh please mention the reason why you didn't add auto-update config.

**Comment by kennynaoh** [comment]
@pvnr0082t ,
I have updated the commit message.
Please help me review it again.
Thanks!

**Comment by pvnr0082t** [comment]
LGTM

**Pull request merged**